apps/system/linux/machine/generic/platform_info.c

```diff
@@ -125,13 +125,29 @@ static void linux_proc_block_set(struct metal_io_region *io,
 	return;
 }
 
+static metal_phys_addr_t linux_proc_offset_to_phys(struct metal_io_region *io,
+						   unsigned long offset)
+{
+	return (offset < metal_io_region_size(io) ? offset : METAL_BAD_OFFSET);
+}
+
+static unsigned long linux_proc_phys_to_offset(struct metal_io_region *io,
+					       metal_phys_addr_t phys)
+{
+	(void)io;
+	return phys;
+}
+
 static struct metal_io_ops linux_proc_io_ops = {
 	.write = NULL,
 	.read = NULL,
 	.block_read = linux_proc_block_read,
 	.block_write = linux_proc_block_write,
 	.block_set = linux_proc_block_set,
 	.close = NULL,
+	/* offset and phys are the same here */
```

Comment:
move this comment in `linux_proc_offset_to_phys` and `linux_proc_phys_to_offset`

Reply:
Done.